src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStringParts = sourceString.split(';');
+  const strWithoutSpaces = sourceStringParts.map((str) => str.trim());
```

Comment:
[CODE KNOWLEDGE] - You don't need to make a copy of an array before each operation. Methods like `map` don't mutate the original array, so you can chain them directly after the `split` method.